docs/HowTo/Deploy/Monitoring-Performance.md

```diff
@@ -71,9 +71,23 @@ block of the `prometheus.yml` file:
 1. Choose **Graph** from the menu bar and click the **Console** tab below.
 
 1. From the **Insert metric at cursor** drop-down, select a metric such as `besu_blockchain_difficulty_total` or
-`besu_blockchain_height` and click **Execute**. The values are displayed below.
+`besu_blockchain_height` and click **Execute**. The values are displayed.
 
-    Click the **Graph** tab to view the data as a time-based graph. The query string is displayed below the graph. 
+    The following table shows metrics that Besu might capture and share with Prometheus.
```

Comment:
This needs rewording slightly - the purpose the EIP was to define the metrics common to Ethereum clients (eg, Besu, Geth, etc) and prefix them with `ethereum_`.  There will also be Besu specific metrics prefixed with `besu_` (we're not listing these specifically at this point because they might still change). 

So it needs to be something like: 
Standard Ethereum metrics are prefixed with `ethereum_` and listed in the table below. Metrics specific to Besu are prefixed with `besu_`.